src/Account/TSPreKeyManager.h

```diff
@@ -26,7 +26,7 @@ typedef NS_ENUM(NSInteger, RefreshPreKeysMode) {
                         success:(void (^)())successHandler
                         failure:(void (^)(NSError *error))failureHandler;
 
-+ (void)refreshPreKeys;
++ (void)checkPreKeys;
```

Comment:
The name "refresh" was misleading; it suggested that this method would always register new prekeys.  This method checks whether we need to register and only registers if necessary.